### ISSUE TITLE
go_struct_tag not working for sqlite

### ISSUE THREAD

**Issue by s12chung** [kyleconroy/sqlc#2285]
### Version

1.17.2

### What happened?

go_struct_tag not working for sqlite

### Database schema

```sql
CREATE TABLE articles (
    id INTEGER PRIMARY KEY
)
```

### Configuration

```yaml
{
  "version": 2,
  "sql": [
    {
      "engine": "sqlite",
      "schema": "query.sql",
      "queries": "query.sql",
      "gen": {
        "go": {
          "out": "db",
          "emit_json_tags": true,
          "overrides": [
            {
              "go_struct_tag": "json:\"-\"",
              "column": "*.id"
            }
          ]
        }
      }
    }
  ]
}
```

### Playground URL

https://play.sqlc.dev/p/61cf06196d3b4d54569ff37d89dfa379197a9a8188bee8243437bfd072a5d101

### What operating system are you using?

macOS

### What type of code are you generating?

Go

**Comment by orisano** [comment]
@s12chung I think it is fixed in v1.19.0.
https://play.sqlc.dev/p/38ab6a11b9f1e05443a4ae1d722bfc20210124443cfc4626e8a243e3781c7854

**Comment by s12chung** [comment]
playground link looks good to me.
